flexmeasures/ui/templates/crud/asset.html

```diff
@@ -236,6 +236,36 @@ <h3>All child assets for {{ asset.name }}</h3>
                 </table>
             </div>
         </div>
+        <div class="row">
+            <div class="copy-url" title="Click to copy URL to clipboard">
+                <script>
+                    function copyUrl() {
+                        if (!window.getSelection) {
+                            alert('Please copy the URL from the location bar.');
+                            return;
+                        }
+                        const dummy = document.createElement('p');
+                        dummy.textContent = window.location.href;
+                        document.body.appendChild(dummy);
+
+                        const range = document.createRange();
+                        range.setStartBefore(dummy);
+                        range.setEndAfter(dummy);
+
+                        const selection = window.getSelection();
+                        // First clear, in case the user already selected some other text
+                        selection.removeAllRanges();
+                        selection.addRange(range);
+
+                        document.execCommand('copy');
+                        document.body.removeChild(dummy);
+                    }
+                </script>
+                <a href="#" onclick="copyUrl()">
```

Comment:
I'd like to get a confirmation (maybe use jquery) that says "URL to this view has been copied to your clipboard."